### PR TITLE
 Added some actual content (and some faked content)

### PR DESCRIPTION
**Pull request by morgantaschuk**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Plugins
-gems:
```
Comment (denis-yuen) [inline _config.yml:2]: Legit change or typo?

File: _data/speakers.yml [hunk 2]
```diff
@@ -1,67 +1,18 @@
 - 
   id: 3
-  name: "Dawid"
-  surname: "Ostrowski"
-  company: "Google Poland"
-  title: "Developer Relations Program Manager"
-  bio: "Dawid is true technology enthusiast, who loves to work in areas where humans and machines meet to create amazing new products and services. He is experienced program manager with strong technical background, who is able to establish priorities for complex projects and execute across multiple organizations, according to long-term objectives. Dawid has worked in Poland, USA, Germany, Belgium and The Netherlands and truly values diversity of international working environment. His most visible characteristics are individualization (when dealing directly with people), ideation&intellection (inexhaustible source of new concepts) and strong communication (experienced trainer, community manager). He is also an activator constantly impatient for action."
-  thumbnailUrl: DawidOstrowski.jpg
+  name: "Katy"
```
Comment (denis-yuen) [inline _data/speakers.yml:3]: :man_shrugging:

File: automation/images/all_image_optimization.bat [hunk 3]
```diff
@@ -1,43 +0,0 @@
-@echo off
```
Comment (denis-yuen) [inline automation/images/all_image_optimization.bat:1]: Aren't these potentially useful for optimizing the site?

**Review by denis-yuen — APPROVED** [review]
A couple deletions from automation that may be worth double-checking
(For this bit https://github.com/gdg-x/zeppelin/wiki/Resources-optimizations#minify-css-and-js )
I'll hook up imgbot to handle the top section in a better way

**Comment by denis-yuen** [comment]
actually I don't have rights to install imgbot https://github.com/marketplace/imgbot but it works pretty well for the first part of what automation is supposed to do

**Comment by morgantaschuk** [comment]
I deleted the automation directory because I got the following email from Github every time I built:


> The page build completed successfully, but returned the following warning for the `master` branch:
> 
> It looks like you're using GitHub Pages to distribute binary files. We strongly suggest that you use releases to ship projects on GitHub. Releases are GitHub's way of packaging and providing software to your users. You can think of it as a replacement to using downloads to provide software. We found the following file(s) which may be a good candidate for releases: automation/images/gifsicle.exe, automation/images/jpegtran.exe, automation/images/optipng.exe, automation/images/pngout.exe, and automation/images/pngquant.exe. For more information, see https://help.github.com/en/articles/about-releases.
> 
> For information on troubleshooting Jekyll see:
> 
>   https://help.github.com/articles/troubleshooting-jekyll-builds
> 
> If you have any questions you can contact us by replying to this email.

File: _config.yml (hunk 1 above)
Comment (morgantaschuk) [reply _config.yml:2]: Yeah, `gems` is deprecated in favour of `plugins`

File: _data/speakers.yml (hunk 2 above)
Comment (morgantaschuk) [reply _data/speakers.yml:3]: ?

**Comment by denis-yuen** [comment]
Yeah, I think `automation/images` can be deleted and replaced with imgbot. 
`automation/minifying` should probably be kept and doesn't seem to part of the above warning

File: _data/speakers.yml (hunk 2 above)
Comment (denis-yuen) [reply _data/speakers.yml:3]: Thought Dawid was already fake, but Googling ... maybe not

**Comment by morgantaschuk** [comment]
I'll make the change.

**Pull request merged**